src/DomainHost.sol

```diff
@@ -73,9 +73,10 @@ abstract contract DomainHost {
     uint256 public rid;         // Remote ordering id
     uint256 public line;        // Remote domain global debt ceiling [RAD]
     uint256 public grain;       // Keep track of the pre-minted DAI in the escrow [WAD]
+    uint256 public ddai;        // A running total of how much is ready for realize as surplus [WAD]
     uint256 public dsin;        // A running total of how much is required to re-capitalize the remote domain [WAD]
-    uint256 public cure;        // The amount of unused debt [RAD]
```

Comment:
A `cure()` function needs to be re-added to supply a `cure` value to the dss `cure.sol`.

Reply:
Actually I'm noticing you purposefully removed the `cure.load()` from the integration tests. I'm assuming this is intentional because it's no longer needed? IE the `tell()` function takes care of everything unless I'm mistaken.

Reply:
Right, sending the not used from the `escrow` to the surplus buffer we would be achieving the same thing than keeping the debt and adding up to the `cure`, that's why I directly removed it.